### DISCUSSION TITLE
Linux newbie stuck with installation on steam deck.

### DISCUSSION THREAD

**Discussion by MrArtyy** [cryinkfly/Autodesk-Fusion-360-for-Linux#332] [Q&A]
Hello!

This is my 2nd attempt to get this working so I am trying to follow the [documentation ](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/wiki/Documentation#downloading-the-installer-package)as closely as possible but, as I am new to Linux, I'm not sure if I'm doing everything right. 

The current problem: When I try to execute "install.sh" it does not appear todo anything, is this normal?

This is what I've done:

- Fresh install of Steam OS version 3.4.8
- Created a folder called “.fusion360” in “Home” folder
- Created a folder called “bin” in “.fusion360” folder
- Download all files from [here ](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux)as a ZIP into the “bin” folder
- Extract  ZIP folder to the same location
- Open Folder > files > builds > stable-branch>bin
- Set “install.sh” to “Is executable”
- Execute “install.sh”
- Does not appear todo anything.
Is this correct? Am I worried about nothing?

Thank you for taking the time to read this and I greatly appreciate any advice you may have for this Linux beginner.

**Comment by NoSadBeHappy** [comment]
Have you ran the sh script in the terminal? do ./install.sh while in the correct directory.
